src/utils/utils.js

```diff
@@ -104,3 +104,38 @@ export const isLoggedInUser = username => {
   }
   return false;
 };
+
+export const isRootPath = pathname => {
+  if (pathname === '/' || pathname === '/search' || pathname === '/sessions') {
```

Comment:
We should consider importing the `menuData` object in `menu.js` instead of hardcoding the root path entrypoints for the dashboard.